travis-setup.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -euo pipefail
+
+# Sets up travis-ci environment for testing bioconda-utils.
+
+if [[ $TRAVIS_OS_NAME = "linux" ]]
+then
+    tag=Linux
+else
+    tag=MacOSX
+fi
+curl -O https://repo.continuum.io/miniconda/Miniconda3-latest-${tag}-x86_64.sh
+sudo bash Miniconda3-latest-${tag}-x86_64.sh -b -p /anaconda
+sudo chown -R $USER /anaconda
+export PATH=/anaconda/bin:$PATH
+
+# TODO: add remaining pip reqs to conda-forge
+conda install -y --file conda-requirements.txt --channel bioconda --channel conda-forge --channel anaconda
```

Comment:
I'd vote for simply using an environment.yml file. We can specify the channels, and both conda and pip dependencies in one place.